packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx

```diff
@@ -90,6 +90,14 @@ const TextInput: typeof TextAreaBase = React.forwardRef(
       };
     });
 
+    React.useEffect(() => {
+      const textarea = document.getElementById(`text-input`);
+
+      if (textarea) {
+        textarea.focus();
+      }
+    }, []);
+
```

Comment:
Is this just mock code that will be replaced later? Seems a bit weird to be finding an element in the document by an ID..

Reply:
Oh yeah can update this one, was mostly focused on the keyboard navigation stuff

Reply:
Fixed, thanks for catching!